### PR TITLE
fundamentals-3.md: add additional resource

### PR DESCRIPTION
**Pull request by AshkanArabim**
Added a link to a W3schools page about JavaScript function parameters that also explains how the 'arguments' object works.

Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [x] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [x] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #XXXXX

**2. This PR:**

**3. Additional Information:**

### CODE REVIEW

**Review by wise-king-sullyman — COMMENTED** [review]
I think that this is a good topic to add an additional resource about, but I wonder if this lesson is an ideal place to add it since we don't cover objects until fundamentals five. I'll think on it and circle back 🙂

**Review by wise-king-sullyman — CHANGES_REQUESTED** [review]
Per my previously discussed reasoning I think this would be a better additional resource in fundamentals 5, can you move it to that lesson?

**Comment by AshkanArabim** [comment]
I removed the additional resource about arguments that I had added before. I will open another pull request to add it to the 'fundamentals 5' section.

**Comment by wise-king-sullyman** [comment]
@AshkanArabim you can just add it to that lesson in this PR if you'd like, a new PR isn't needed.

**Comment by AshkanArabim** [comment]
I made the requested changes.
I noticed that in fundamentals-5.md, unlike the others, the additional resources section was after the knowledge check section. I made the correction.

**Review by wise-king-sullyman — CHANGES_REQUESTED** [review]
Can you actually move the additional resources back to below the knowledge check section? That's the layout that we're going for in the future per [our lesson template](https://github.com/TheOdinProject/curriculum/blob/main/templates/lesson-template.md).

After that this should be good to go! 🙂

File: foundations/javascript_basics/fundamentals-5.md [hunk 1]
```diff
@@ -50,4 +50,4 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-*   It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
\ No newline at end of file
+*   Learn about the 'arguments' object on [this page](https://www.w3schools.com/js/js_function_parameters.asp) about function parameters.
```
Comment (wise-king-sullyman) [inline foundations/javascript_basics/fundamentals-5.md:53]:
```suggestion
*   Learn about the 'arguments' object on this [page about function parameters](https://www.w3schools.com/js/js_function_parameters.asp).
```

**Review by wise-king-sullyman — CHANGES_REQUESTED** [review]
Actually I have one final tweak I'd like to see, just a slight update of the link text used to keep things more accessible.

You can read about the reasoning behind this suggestion in this great [article about link text from Harvard](https://accessibility.huit.harvard.edu/technique-writing-link-text).

**Review by wise-king-sullyman — APPROVED** [review]
Looks perfect, thanks!

**Pull request merged**